src/styles/main.scss

```diff
@@ -0,0 +1,7 @@
+@import "./blocks/reset.scss";
+
+@import "./blocks/centre.scss";
+@import "./blocks/minutes.scss";
+@import "./blocks/seconds.scss";
+
+@import "./utils/keyframes";
```

Comment:
Where I can see those files?

Reply:
Looks like you didn't add those files to commit, try to add them